packages/@react-spectrum/s2/src/TableView.tsx

```diff
@@ -784,20 +790,29 @@ function ResizableColumnContents(props: ResizableColumnContentProps) {
           </div>
           <Chevron size="M" className={chevronIcon} />
         </Button>
-        <Menu onAction={onMenuSelect} items={items} styles={style({minWidth: 128})}>
-          {(item) => <MenuItem>{item?.label}</MenuItem>}
+        <Menu onAction={onMenuSelect} styles={style({minWidth: 128})}>
+          {items.length > 0 && (
+            <MenuSection aria-label='Sort or resize?'>
+              <Collection items={items}>
+                {(item) => <MenuItem>{item?.label}</MenuItem>}
+              </Collection>
```

Comment:
best way to add other sort options here (for other columns like doc cloud has?)

Reply:
Discussion so far, this is ok for the different/ other sorts to be separate, they are likely not as closely related